sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,21 @@
+UPDATE `creature_text` SET `Probability` = 25 WHERE `CreatureID` = 7800 AND `GroupID` = 1;
+UPDATE `creature_template` SET `mechanic_immune_mask`=650854271, `flags_extra` = 256 WHERE `entry`=7915;
+UPDATE `creature_template` SET `flags_extra` = `flags_extra` | 256 WHERE `entry`=7800;
+
+
+DELETE FROM `smart_scripts` WHERE `source_type`=0 AND `entryorguid` IN (7800,7915);
+UPDATE `creature_template` SET `ScriptName`='boss_mekgineer_thermaplugg', `AIName`='' WHERE entry=7800;
+UPDATE `creature_template` SET `ScriptName`='npc_walking_bomb', `AIName`='' WHERE entry=7915;
+
+UPDATE `gameobject_template` SET `ScriptName`='go_button' WHERE `entry` IN (142214,142215,142216,142217,142218,142219);
+UPDATE `gameobject_template` SET `ScriptName`='go_gnome_face' WHERE `entry` IN (142211,142210,142209,142208,142213,142212);
+
+-- SPELL Activate Bomb A (11511)
+-- SPELL Activate Bomb B (11795)
+DELETE FROM `spell_target_position` WHERE `ID` IN (11511, 11795);
+INSERT INTO `spell_target_position` (`ID`, `EffectIndex`, `MapID`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`) VALUES
+(11511, 0, 90, -531.14, 670.136, -310.0, 0.0),
+(11795, 0, 90, -531.14, 670.136, -310.0, 0.0);
+
+DELETE FROM `spell_script_names` WHERE `spell_id` = 10101;
```

Comment:
TC codestyle dictates deleting spells from `spell_script_names` by `ScriptName`, i.e.
```sql
DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_10101_knock_back';
```
Is there a good reason that this should not be the case here?

Reply:
I renamed the spell to something generic, since knock away with a threat drop is common.
https://wotlk.evowow.com/?spell=10101#see-also
Possibly up to 8 spells that should all behave the same and can share this script.

The obvious case for this is 
https://github.com/TrinityCore/TrinityCore/blob/master/src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/boss_broodlord_lashlayer.cpp#L90-L95

When/if this PR is merged, I'm going to make a new PR fixing Broodlord to use the generic spellscript. Would you then suggest I do?
```sql
DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_gen_knock_away_w_threat_drop';
INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (25778, 'spell_gen_knock_away_w_threat_drop');
```

Do you see the problem here?

Reply:
No, that rule is dumb and should be done away with.
That "solution" is just causing extra unnecessary work, don't make life harder on yourself.

Reply:
Not to mention the issues it can cause with order of branch merges.

Reply:
Fair enough, those are valid points too.